crates/net/src/http/method.rs

```diff
@@ -0,0 +1,86 @@
+use std::{fmt::Display, str::FromStr};
+
+/// HTTP methods that can be used in a fetch request.
+/// The methods as defined by the [fetch spec](https://fetch.spec.whatwg.org/#methods).
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum Method {
```

Comment:
I prefer how the http crate handled it. We should use a struct with associated constants and not enum as you can choose your own http verb.

See: https://github.com/rustwasm/gloo/pull/312#discussion_r1148362780

Reply:
I'm not so sure if that's a good idea. Custom HTTP verbs (as far as I know) are not standard. Does fetch even support them?

Reply:
Just decide what is the better way, I can make the change.